include/jemalloc/internal/tsd_structs.h

```diff
@@ -69,6 +68,7 @@ struct tsdn_s {
 };
 
 static const tsd_t tsd_initializer = TSD_INITIALIZER;
+UNUSED static const void *malloc_tsd_no_cleanup = (void *)0;
```

Comment:
As mentioned previously, this change is to avoid a compiler issue, where the empty function version before this diff cannot be compared correctly in "a_cleanup != malloc_tsd_no_cleanup" under Windows MinGW64.